UI/static/css/index.css

```diff
@@ -0,0 +1,233 @@
+body {
+    /* background: linear-gradient(to right, red, blue); */
+    background: linear-gradient(45deg, #13547a 0%, #438b9c 50%, #80d0c7 100%);
+}
+
+.grid {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-areas: "navigation" "attention" "self-selection" "testimonial-heading" "testimonials" "product" "action-call" "footer";
+    grid-template-rows: auto 300px 300px 50px 200px 1fr 100px 200px;
+}
+
+.navigation {
+    grid-area: navigation;
+    justify-self: center;
+}
+
+.attention {
+    grid-area: attention;
+    background-image: url("../images/taxi1.jpeg");
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    opacity: 0.8;
+    display: flex;
+    align-items: flex-end;
+}
+
+.attention-call {
+    grid-area: attention-call;
+    justify-self: center;
+    width: 100%;
+    /* position: absolute; */
```

Comment:
Remove commented out code before committing

Reply:
wonderful UI

Reply:
Great UI. Looks good

My only concern is related to version control, I've noticed that you've merged into master.
Check this out [Feature-branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)

Reply:
will check, thanx